_layouts/home.html

```diff
@@ -3,24 +3,28 @@
 ---
 
 <div class="home">
+  {% if page.title %}
+    <h1 class="page-heading">{{ page.title }}</h1>
+  {% endif %}
 
   {{ content }}
 
-  <h1 class="page-heading">Posts</h1>
-  
-  <ul class="post-list">
-    {% for post in site.posts %}
-      <li>
-        {% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
-        <span class="post-meta">{{ post.date | date: date_format }}</span>
+  {% if site.posts.size > 0 %}
+  <h2 class="post-list-heading">{{ page.list_title | default: "Posts" }}</h2>
```

Comment:
Similar question here. When would you want this to be something other than Posts since the list is hard coded to the Posts collection (and a use sophisticated enough to introduce their own second collection would likely want to modify the template anyway)?